middleware/envvar/envvar.go

```diff
@@ -0,0 +1,80 @@
+package envvar
+
+import (
+	"github.com/gofiber/fiber/v2"
+	"github.com/gofiber/fiber/v2/utils"
+	"os"
+	"strings"
+)
+
+// Config defines the config for middleware.
+type Config struct {
+	// Path specifies the path of handler that exposes environment variables
+	Path string
```

Comment:
specifying the path is not necessary in my opinion
you can say outside where the middleware should run

```go
app.Use("/expose/envvars", envvar.New(envvar.Config{
    ...
}))
```

Reply:
So we should not support app.Use(envvars.New()) approach? User must provide path?

Reply:
Like monitor middleware